### PR TITLE
fixed linting error merged from vets-website

### PR DESCRIPTION
**Pull request by timwright12**

### CODE REVIEW

**Review by jhonnyoddball — APPROVED** [review]

File: src/site/stages/build/process-cms-exports/transformers/paragraph-staff_profile.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const { getImageCrop } = require('./helpers');
 
 const setImageCrop = media => {
-  const imageObj = Object.assign({}, media);
+  const imageObj = { ...media };
```
Comment (jhonnyoddball) [inline src/site/stages/build/process-cms-exports/transformers/paragraph-staff_profile.js:4]: Why is this not a linting problem in `vets-website`?
Do we need to fix it too?

Comment (timwright12) [reply src/site/stages/build/process-cms-exports/transformers/paragraph-staff_profile.js:4]: @jhonnyoddball no idea, linting issues really blew up when we started porting stuff into this repo, almost like they were working off different linting rules

Comment (jhonnyoddball) [reply src/site/stages/build/process-cms-exports/transformers/paragraph-staff_profile.js:4]: Maybe we need to align them? or point content-build  to `vets-website`'s `.eslintrc` file?

Comment (timwright12) [reply src/site/stages/build/process-cms-exports/transformers/paragraph-staff_profile.js:4]: yeah, I'll make a task to investigate that

**Pull request merged**